core/modules/node/node.module

```diff
@@ -1863,12 +1859,14 @@ function node_block_view($delta = '', $settings = array()) {
 
   switch ($delta) {
     case 'syndicate':
-      $block['subject'] = t('Syndicate');
-      $block['content'] = array(
-        '#theme' => 'feed_icon',
-        '#url' => 'rss.xml',
-        '#title' => t('Syndicate'),
-      );
+      if (backdrop_valid_path('rss.xml')) {
+        $block['subject'] = t('Syndicate');
+        $block['content'] = array(
+          '#theme' => 'feed_icon',
+          '#url' => 'rss.xml',
```

Comment:
Long-term, perhaps we make this an option instead of hard-coding the path `rss.xml`. Perhaps show a drop-down of Views that have RSS feeds? Not in this issue though.